### ISSUE TITLE
Requirements to be an I-D

### ISSUE THREAD

**Issue by anewton1998** [SIDN/ietf-wg-rpp-charter#25]
From James Gould:

> Reuse language from the DELEG charter with “The RPP working group will first document the requirements for RPP that a functional equivalent or superset of EPP.  The initial version of the requirements document should have broad general consensus and must be adopted by the WG before work on the solution documents begins.”
a. Ensure that RPP meets the functional requirements that are currently met by EPP. 
b. This can proceed the first sentence of the Deliverables section “This working group will find consensus…”.

**Comment by anewton1998** [comment]
#18

**Comment by mwullink** [comment]
I agree that just because DELEG has chosen to publish the requirements as an ID should not mean that RPP must do the same.

Both are very different working groups, both in scope. goals and solution space.

i would prefer a more agile approach whereby we use the wiki and are able to update the requirements when this becomes desirable and only after wg consensus.

**Comment by pawel-kow** [comment]
Capture from the E-mail thread:

>> [PK] I think we had the discussion before[ https://mailarchive.ietf.org/arch/msg/rpp/3H_h8mLz9Rod7o_X8e1zEdvqv88/](https://mailarchive.ietf.org/arch/msg/rpp/3H_h8mLz9Rod7o_X8e1zEdvqv88/) and eventually there seemed to be little value of bringing it to RFC level that would constrain the process and change control of the requirements. IESG's Statement on Support Documents gives us this opportunity and IMHO the WG to be formed should make a good use of it.
>>

> [JG2] I believe that the requirements should be captured in an informational RFC, like what has been done with other working groups such as what’s defined in the DELEG charter.

[PK2] The IESG statement seems to be clear when such working documents shall become an RFC. I don't think we need an IETF-wide consensus on requirements. WG consensus written in the charter shall be just the right balance.

**Comment by jasdips** [comment]
One way to consider the need for a requirements RFC, versus simply publishing requirements on the WG wiki, for RPP is whether replacing EPP, an Internet standard, with RPP in the future deserves an IETF-wide consensus or not?

(Since DNS, like EPP, is so widely used, wonder if the requirements RFC route for DELEG was picked so as to more formally clarify the evolution need vis-a-vis NS.)

**Comment by shollenbeck** [comment]
> One way to consider the need for a requirements RFC, versus simply publishing requirements on the WG wiki, for RPP is whether replacing EPP, an Internet standard with a wide deployment base, with RPP in the future deserves an IETF-wide consensus or not?
> 
> (Since DNS, like EPP, is so widely used, wonder if the requirements RFC route for DELEG was picked so as to more formally clarify the evolution need vis-a-vis NS.)

This is a good point. If we assume that "replace EPP" is not a current goal, we can also assume that there's less of a need to gain IETF consensus on a requirements document.

**Comment by pawel-kow** [comment]
> This is a good point. If we assume that "replace EPP" is not a current goal, we can also assume that there's less of a need to gain IETF consensus on a requirements document.

+1

**Comment by jasdips** [comment]
Isn't RPP intended to replace EPP in the long run?

**Comment by pawel-kow** [comment]
It was mentioned in the BoF meeting that this is not an objective.So far I see no reason for IETF to say that EPP is bad, stop using it and use something different instead. It will be with us the same as ipv4 is with us parallel to ipv6.It was a different story with WHOIS and RDAP where clearly there was a reason to replace WHOIS and state it as a goal.Viele Grüße / Kind regardsPawel Kowalik Am 14.11.2024 um 00:00 schrieb Jasdip Singh ***@***.***>:﻿
Isn't RPP intended to replace EPP in the long run?

**Comment by jasdips** [comment]
Ack

**Comment by anewton1998** [comment]
I am not seeing much support for this idea and therefore closing this issue.
